compiler/include/expr.h

```diff
@@ -203,8 +203,9 @@ class CallExpr : public Expr {
   Expr*           get(int index);
   FnSymbol*       findFnSymbol();
 
-  bool            isPrimitive(PrimitiveTag primitiveTag);
-  bool            isPrimitive(const char*  primitiveName);
+  bool            isPrimitive()                                          const;
+  bool            isPrimitive(PrimitiveTag primitiveTag)                 const;
+  bool            isPrimitive(const char*  primitiveName)                const;
```

Comment:
I wonder the same thing.   I am unsure how to think about maintaining consistency
between Master and string-as-rec ahead of the eventual merge.



On Sep 18, 2015, at 12:47 PM, Lydia Duncan <notifications@github.com<mailto:notifications@github.com>> wrote:


In compiler/include/expr.h<https://github.com/chapel-lang/chapel/pull/2565#discussion_r39893678>:

> @@ -203,8 +203,9 @@ class CallExpr : public Expr {
>    Expr*           get(int index);
>    FnSymbol*       findFnSymbol();
>
> -  bool            isPrimitive(PrimitiveTag primitiveTag);
> -  bool            isPrimitive(const char*  primitiveName);
> +  bool            isPrimitive()                                          const;
> +  bool            isPrimitive(PrimitiveTag primitiveTag)                 const;
> +  bool            isPrimitive(const char*  primitiveName)                const;


Should this change be applied to master? (post release, of course)

—
Reply to this email directly or view it on GitHub<https://github.com/chapel-lang/chapel/pull/2565/files#r39893678>.